### PR TITLE
Fixed Updating of Created on of Developer Service on refresh

### PR DESCRIPTION
**Pull request by ruddi10**
I have done a minor change in the ApplicationDetailSerializer , removed datetime_created field from exclude and added to read_only_fields

### CODE REVIEW

File: omniport/core/open_auth/serializers/application.py [hunk 1]
```diff
@@ -99,9 +99,9 @@ class Meta:
             'client_id',
             'client_secret',
             'authorization_grant_type',
+             'datetime_created'
```
Comment (algomaster99) [inline omniport/core/open_auth/serializers/application.py:102]: Remove the extra space

**Review by algomaster99 — CHANGES_REQUESTED** [review]
@ruddi10 nice work
Refer to the instructions in your other PR on how to write commit messages. Otherwise look at previous commits in this repository.

**Comment by ruddi10** [comment]
Made the changes you asked for.Thanks

File: omniport/core/open_auth/serializers/application.py [hunk 2]
```diff
@@ -99,9 +99,9 @@ class Meta:
             'client_id',
             'client_secret',
             'authorization_grant_type',
+            'datetime_created'
```
Comment (algomaster99) [inline omniport/core/open_auth/serializers/application.py:102]: Add a trailing comma

**Review by algomaster99 — CHANGES_REQUESTED** [review]
last change 😅

File: omniport/core/open_auth/serializers/application.py (hunk 2 above)
Comment (ruddi10) [reply omniport/core/open_auth/serializers/application.py]: Done kindly review

**Pull request merged**